docs/custom-image.md

```diff
@@ -0,0 +1,66 @@
+# Using a custom image
+
+Pilot hubs use an image we curate as the default. This can be replaced with your
+own custom image fairly easily. However, custom images should be maintained
+by the hub admins, and we won't be able to help much with things in it.
```

Comment:
can we define who the "hub admin" is? Our official roles that we define are "hub engineer", "community champion", and "hub architect". We should make it clear which of those roles is "the hub admin" since I suspect that the hub engineer will also have admin rights on the hubs